eledmac.dtx

```diff
@@ -1772,8 +1772,8 @@
 % The default series order is the one called with the \verb+series+ option of the package, or, if this option is not used, A, B, C, D, E, Z. 
 % Series order determines footnotes order.
 %
-% \DescribeMacro{seriesatbegin}\DescribeMacro{seriesatend} However in some specific cases, you need to change the series order inside the document.
-% You can use \cs{seriesatbegin}\marg{<s>} to push the series \verb+<s>+ at the beginning, and \cs{seriesatend}\marg{<s>}  to push the series \verb+<s>+ at the end.
+% \DescribeMacro{seriesatbegin}\DescribeMacro{seriesatend} However in some specific cases, you need to change the series order at some point inside the document.
+% You can use \cs{seriesatbegin}\marg{<s>} to pull a given series \verb+<s>+ at the beginning, and \cs{seriesatend}\marg{<s>}  to push it \verb+<s>+ at the end.
```

Comment:
On Sun, Apr 12, 2015 at 02:37:54PM -0700, Ma�eul Rouquette wrote:
> > @@ -1772,8 +1772,8 @@
> >  % The default series order is the one called with the \verb+series+ option of the package, or, if this option is not used, A, B, C, D, E, Z. 
> >  % Series order determines footnotes order.
> >  %
> > -% \DescribeMacro{seriesatbegin}\DescribeMacro{seriesatend} However in some specific cases, you need to change the series order inside the document.
> > -% You can use \cs{seriesatbegin}\marg{<s>} to push the series \verb+<s>+ at the beginning, and \cs{seriesatend}\marg{<s>}  to push the series \verb+<s>+ at the end.
> > +% \DescribeMacro{seriesatbegin}\DescribeMacro{seriesatend} However in some specific cases, you need to change the series order at some point inside the document.
> > +% You can use \cs{seriesatbegin}\marg{<s>} to pull a given series \verb+<s>+ at the beginning, and \cs{seriesatend}\marg{<s>}  to push it \verb+<s>+ at the end.
> 
> je parle ici d'une s�rie sp�cifique <s>. L� du coup on ne sait pas ce que d�signe le "it" non?
> 
Si, parce que j'ai ajout� avant: "to pull a given series", une s�rie
donn�e. Du coup, "it" renvoie � cette s�rie donn�e.